### PR TITLE
DTSPB-695 Add new page in GoP journey with new fields

### PR DESCRIPTION
**Pull request by KleoG**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPB-695

### Change description ###
Add new page in GoP journey with new fields

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/probate/functional/businessvalidation/SolCcdServiceBusinessValidationTests.java [hunk 1]
```diff
@@ -500,6 +500,19 @@ public void shouldTransformCaseWithCitizenAttributes() {
         assertEquals("1", registrySequenceNumber);
     }
 
+    @Test
+    public void shouldTransformCaseWithTrustCorpAttributes(){
+        String response = transformCase("success.TrustCorpAttribtesSaved.json", TRANSFORM_URL);
+
```
Comment (vinceholt) [inline src/functionalTest/java/uk/gov/hmcts/probate/functional/businessvalidation/SolCcdServiceBusinessValidationTests.java:506]: COULD: typo success.TrustCorpAttribtesSaved.json, tho it does match the filename

File: src/test/end-to-end/pages/solicitorApplyProbate/grantOfProbate/page3.js [hunk 2]
```diff
@@ -1,12 +1,48 @@
...
+        await I.click(`#solsAdditionalExecutorList_0_additionalExecNameOnWill-${grantOfProbateConfig.optionYes}`);
+        await I.click(`#solsAdditionalExecutorList_0_additionalApplying-${grantOfProbateConfig.optionYes}`);
+
+        await I.waitForElement('#solsAdditionalExecutorList_0_additionalExecAddress_additionalExecAddress_postcodeInput', testConfig.TestTimeToWaitForText);
+        await I.fillField('#solsAdditionalExecutorList_0_additionalExecAddress_additionalExecAddress_postcodeInput', grantOfProbateConfig.page2_executorPostcode);
+        await I.click('#solsAdditionalExecutorList_0_additionalExecAddress_additionalExecAddress > div  > div > button');
+        await I.waitForElement('#solsAdditionalExecutorList_0_additionalExecAddress_additionalExecAddress_addressList', testConfig.TestTimeToWaitForText);
+        await I.fillField('#solsAdditionalExecutorList_0_additionalExecAliasNameOnWill', grantOfProbateConfig.page2_executorAliasName);
+        await I.retry(10).selectOption('#solsAdditionalExecutorList_0_additionalExecAddress_additionalExecAddress_addressList', grantOfProbateConfig.page2_executorAddress);
+    }
```
Comment (vinceholt) [inline src/test/end-to-end/pages/solicitorApplyProbate/grantOfProbate/page3.js:45]: COULD: retrying 10 times feels a bit implicit, wouldn't a change to waitForEnabled from waitForElement  #solsAdditionalExecutorList_0_additionalExecAddress_additionalExecAddress_addressList do the trick, perhaps with a wait for the option too? Happy if it works tho as the key thing is it doesn't cause unnecessary waits.

**Review by vinceholt — APPROVED** [review]
Have approved, but left a couple of possible COULDs that you might want to look at, or they can be left if you prefer as is

File: src/functionalTest/java/uk/gov/hmcts/probate/functional/businessvalidation/SolCcdServiceBusinessValidationTests.java (hunk 1 above)
Comment (KleoG) [reply src/functionalTest/java/uk/gov/hmcts/probate/functional/businessvalidation/SolCcdServiceBusinessValidationTests.java:506]: Good spot, made the change now

**Pull request closed without merging**